src/app/store/machine/slice.ts

```diff
@@ -1228,6 +1228,22 @@ const machineSlice = createSlice({
       state.filtersLoading = false;
       state.filtersLoaded = true;
     },
+    cleanupRequest: {
```

Comment:
I think this should probably be something like `cleanupDetailsRequest` as we'll have details, list and count requests that need to be cleaned up, or if you're imagining this will clean up any request then maybe below it should be:

```
if (action.meta.callId && action.meta.callId in state.details) {
  delete state.details[action.meta.callId];
}
```